code/game/machinery/doors/airlock_control.dm

```diff
@@ -39,8 +39,7 @@
 	if(command_completed(cur_command))
 		cur_command = null
 	else
-		if(!isprocessing)
-			START_PROCESSING(SSmachines, src)
+		START_PROCESSING(SSmachines, src)
```

Comment:
Note to self, what in the same heck is this for; doors are supposed to run on timers these days.